code/parse/sexp.cpp

```diff
@@ -564,6 +564,7 @@ SCP_vector<sexp_oper> Operators = {
 	{ "tech-add-weapons",				OP_TECH_ADD_WEAPON,						1,	INT_MAX,	SEXP_ACTION_OPERATOR,	},
 	{ "tech-add-intel",					OP_TECH_ADD_INTEL,						1,	INT_MAX,	SEXP_ACTION_OPERATOR,	},	// Goober5000
```

Comment:
Since it's just a few lines of code, I suggest a `tech-remove-intel` sexp be added as well.